### DISCUSSION TITLE
request help: How to hot update the plugins' code?

### DISCUSSION THREAD

**Discussion by zgLiuBing** [apache/apisix#5216] [Q&A]
### Issue description

现在是先执行make stop在执行make run才能看到效果
有什么办法改完代码就可以看到效果吗
lua_code_cahe这个是实现lua热更新的吗
lua_code_cahe的配置步骤是什么？

### Environment

- apisix version (cmd: `apisix version`):
- OS (cmd: `uname -a`):
- OpenResty / Nginx version (cmd: `nginx -V` or `openresty -V`):
- etcd version, if have (cmd: run `curl http://127.0.0.1:9090/v1/server_info` to get the info from server-info API):
- apisix-dashboard version, if have:
- the plugin runner version, if the issue is about a plugin runner (cmd: depended on the kind of runner):
- luarocks version, if the issue is about installation (cmd: `luarocks --version`):

**Comment by tzssangglass** [comment]
> 现在是先执行make stop在执行make run才能看到效果
> 有什么办法改完代码就可以看到效果吗

see: https://github.com/apache/apisix/blob/master/docs/en/latest/architecture-design/plugin.md#hot-reload

> lua_code_cahe这个是实现lua热更新的吗

yes

> lua_code_cahe的配置步骤是什么？

see: https://moonbingbing.gitbooks.io/openresty-best-practices/content/ngx_lua/hot_load.html
